py/picca/delta_extraction/astronomical_objects/desi_forest.py

```diff
@@ -77,9 +73,6 @@ class DesiForest(Forest):
     ivar: array of float (from Forest)
     Inverse variance
 
-    lambda_: array of float or None (from Forest)
-    Wavelength (in Angstroms)
-
     log_lambda: array of float or None (from Forest)
     Logarithm of the wavelength (in Angstroms)
```

Comment:
Maybe add a few extra words to differentiate log_lambda from log_lambda_grid. I'm guessing that this array is shorter, since it only has some of the values in log_lambda_grid?

Reply:
See also the commit on updated docstrings